Visualization/AFW_Display_Demo.ipynb

```diff
@@ -0,0 +1,303 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# **Demo of lsst.afw.display -- displaying images using the LSST DM Astronomical Framework library**\n",
+    "\n",
+    "**Owner:** Brant Robertson ([@brantr](https://github.com/LSSTScienceCollaborations/DMStackClub/issues/new?body=@brantr))  \n",
```

Comment:
We should make sure all our URLs say "StackClub" and not "DMStackClub"

Reply:
Fixed, thanks.